README.md

```diff
@@ -108,6 +108,7 @@ These Firefox extensions can help prevent connections to Google domains and also
   - [Subscribe to a (local) newspaper](https://en.wikipedia.org/wiki/Category:Lists_of_newspapers_by_country) and support independent and qualitative journalism.
   - Removed Reddit news as they are typically *extremely* biased to the general populace.
   - [AllSides](https://www.allsides.com/unbiased-balanced-news) - Seemingly unbiased news source. Thanks @jeanofthedead
+  - Use good old-fashioned RSS Feeds
```

Comment:
This is a good point, minus the problem that you would still need to navigate to a certain privacy-invasive news site in order to locate their RSS feed.

With that said, I am still willing to add it if you're willing to provide some privacy friendly RSS readers (I personally use [Feeder](https://f-droid.org/app/com.nononsenseapps.feeder) on Android using the F-Droid app). If you have another Android or iOS suggestion or a desktop suggestion I'd love to see it. If you can't, that is fine and I can add it tomorrow but right now I'm about to get to sleep.

Reply:
Added Feeder